### PR TITLE
PrestoDB use inject

### PR DESCRIPTION
**Pull request by humengyu2012**

### CODE REVIEW

**Comment by humengyu2012** [comment]
I changed the packaging method and modified the `pom.xml` file extensively. I'm not sure if this is a good approach. Could you please take a look? @JingsongLi  I've documented the new packaging method in the `README.md`.

**Comment by feloxx** [comment]
LGTM

**Comment by JingsongLi** [comment]
You need to modify test dependencies too to make test passing.

**Comment by humengyu2012** [comment]
> You need to modify test dependencies too to make test passing.

Sorry, I will fix the tests later.

**Comment by humengyu2012** [comment]
rebase main.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,77 @@ This repository is Presto Connector for the [Apache Paimon](https://paimon.apach
...
 
 Apache Paimon is an open source project of [The Apache Software Foundation](https://apache.org/) (ASF).
+
+## Getting Started
+
+### Build
+
+| Version         | Command                                                     |
+|-----------------|-------------------------------------------------------------|
+| [0.236, 0.268)  | `mvn clean install -DskipTests -am -pl paimon-presto-0.236` |
```
Comment (JingsongLi) [inline README.md:15]: If we have this, do we need to have paimon-presto-0.236 and 0.238?

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**